<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Searchform</class>
 <widget class="QWidget" name="Searchform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>搜索</string>
  </property>
  <widget class="QPushButton" name="search_bt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QComboBox" name="search_tar_cb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_le">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>489</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="search_ec">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>68</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索对象：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
